### ISSUE TITLE
small fix: add macro's value upon hovering on them

### ISSUE THREAD

**Issue by tweenko** [YoYoGames/GameMaker-Bugs#6632]
### Is your feature request related to a problem?

my project is kind of big and has a lot of folders, so to view a value of a macro i have to browse through them to find it, which is kind of annoying.

### Describe the solution you'd like

add macro's value upon hovering on them

### Describe alternatives you've considered

or make the middle mouse button redirect you to the script where the macro is located

**Comment by glebtsereteli** [comment]
Feather can do that already.
![image](https://github.com/user-attachments/assets/619468cd-d502-4ca2-b311-922b7ecd0b71)

**Comment by tweenko** [comment]
whoops i meant something else hold on i meant enumenators

**Comment by YYDan** [comment]
Yeah, as shown above, this is already supported on macros, but enums do not do the same. Have added extra info into the request now.

**Comment by tweenko** [comment]
thank you!
